libs/community/pyproject.toml

```diff
@@ -100,6 +101,7 @@ anthropic = "^0.3.11"
 fireworks-ai = "^0.9.0"
 vdms = "^0.0.20"
 exllamav2 = "^0.0.18"
+pyarrow = "^17.0.0"
```

Comment:
Could you remove changes from pyproject toml and revert the poetry lock file? Instead you can the extended testing requirements file.